cfgov/cfgov/settings/base.py

```diff
@@ -36,25 +38,45 @@
 PASSWORD_HASHERS = global_settings.PASSWORD_HASHERS
 
 # Application definition
-
 INSTALLED_APPS = (
     'permissions_viewer',
-    'wagtail.wagtailcore',
-    'wagtail.wagtailadmin',
-    'wagtail.wagtaildocs',
-    'wagtail.wagtailsnippets',
-    'wagtail.wagtailusers',
-    'wagtail.wagtailimages',
-    'wagtail.wagtailembeds',
-    'wagtail.contrib.wagtailfrontendcache',
-    # 'wagtail.wagtailsearch',  # TODO: conflicts w/ haystack, need to revisit
-    'wagtail.wagtailredirects',
-    'wagtail.wagtailforms',
-    'wagtail.wagtailsites',
+)
 
+if wagtail.VERSION >= (2, 0):
+    INSTALLED_APPS += (
+        'wagtail.core',
+        'wagtail.admin',
+        'wagtail.documents',
+        'wagtail.snippets',
+        'wagtail.users',
+        'wagtail.images',
+        'wagtail.embeds',
+        'wagtail.contrib.frontend_cache',
+        'wagtail.contrib.redirects',
+        'wagtail.contrib.forms',
+        'wagtail.sites',
+        'wagtail.contrib.routable_page',
+    )
+else:
+    INSTALLED_APPS += (
+        'wagtail.wagtailcore',
+        'wagtail.wagtailadmin',
+        'wagtail.wagtaildocs',
+        'wagtail.wagtailsnippets',
+        'wagtail.wagtailusers',
+        'wagtail.wagtailimages',
+        'wagtail.wagtailembeds',
+        'wagtail.contrib.wagtailfrontendcache',
+        # 'wagtail.search',  # TODO: conflicts w/ haystack, need to revisit
+        'wagtail.wagtailredirects',
+        'wagtail.wagtailforms',
+        'wagtail.wagtailsites',
+        'wagtail.contrib.wagtailroutablepage',
```

Comment:
unfortunately the name changed in two ways for routable page, to this:

```suggestion
        'wagtail.contrib.routable_page',
```

Reply:
### Whoops -- disregard. Of course this is in the else section.